spot-ml/src/main/scala/org/apache/spark/sql/WideUDFs.scala

```diff
@@ -16,8 +16,8 @@
  */
```

Comment:
do we still need this file with spark 2.1 and scala 2.11 ?

The whole point was that in older versions, we needed to add definitions for udfs with lots of arguments -- perhaps spark  now handles these.  it's worth a look...